src/lib/security/selinux_config.rb

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) [2021] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "yast2/execute"
+
+module Security
+  # Class for handling SELinux kernel params
+  class SelinuxConfig
+    include Yast::Logger
+    Yast.import "Bootloader"
+
+    attr_reader :policy
+
+    def initialize
+      propose_policy if Yast::Mode.installation
+      @policy = configured_policy
+    end
+
+    # Returns the policy applied in the running system
+    def running_policy
+      Yast::Execute.locally!(GETENFORCE_PATH, stdout: :capture).chomp.downcase.to_sym
+    rescue Cheetah::ExecutionFailed
```

Comment:
this can actually hide other error states. So I would not use debug unless exit code is 127

Reply:
>  So I would not use debug unless exit code is 127

Did you mean _:disabled_ instead _debug_?

Reply:
I mean that you can consider missing tool as :disabled, but you rescue all error states including e.g. 1 or 2 which can mean something different

Reply:
Finally I decided to simple log the exception message, see https://github.com/yast/yast-security/pull/83/files#diff-0a09e81c830190bb29f23761529237b497379394ad66db6a78b8d671f49791c3R63-R70. We can change it later if we still thinking it is really needed.